.gitignore

```diff
@@ -2,3 +2,7 @@ _site
 .sass-cache
 .jekyll-metadata
 *.out
+
+# direnv, posh-direnv
```

Comment:
What's the purpose of these additions?

Reply:
So people can use these tools, whose configs are easiest to keep in the repo but should not be checked in:
https://direnv.net/ or https://github.com/takekazuomi/posh-direnv/blob/master/README.rst